bin/run-ldc93s1.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/sh
```

Comment:
This one was useful for debugging people's setting up issues

Reply:
Yes, please keep run-ldc93s1.sh, I use it for development all the time.